guides/v2.2/rest/retrieve-filtered-responses.md

```diff
@@ -1,18 +1,14 @@
 ---
 group: rest
-subgroup: Web APIs
-title: Retrieve filtered responses for REST APIs
-menu_title: Retrieve filtered responses
-menu_order: 3
-version: 2.0
-redirect_from: /guides/v2.0/howdoi/webapi/filter-response.html
+title: Retrieve filtered responses for REST endpoints
+version: 2.2
```

Comment:
Dima has an unmerged PR that removes the `version` tag, so you may need to delete this later.

https://github.com/magento/devdocs/pull/2750